tip-payment/Anchor.toml

```diff
@@ -22,7 +22,6 @@ members = [
     "programs/tip-distribution",
     "programs/tip-payment"
 ]
-exclude = ["programs/merkle-tree"]
```

Comment:
@esemeniuc @0xEdgar double checking this is safe to delete? I don't see any usage here or in jito-solana

Reply:
yea that looks like its gone

Reply:
yea jito-solana has its own copy: https://github.com/jito-foundation/jito-solana/blob/master/merkle-tree/src/merkle_tree.rs